### PR TITLE
[TfLite] SPLIT_V quantization support for int8

### PR DESCRIPTION
**Pull request by wwwind**
This PR adds int8 quantization for SPLIT_V operator. See issue https://github.com/tensorflow/tensorflow/issues/36965
I added tests as for split_v for int8 by refactoring the existing one, so that they are not Copy&Paste.

### CODE REVIEW

File: tensorflow/lite/tools/optimize/operator_property.cc [hunk 1]
```diff
@@ -92,6 +92,12 @@ OperatorProperty GetOperatorProperty(const ModelT* model, int subgraph_index,
       property.restrict_same_input_output_scale = true;
       property.version = 2;
       break;
+    case BuiltinOperator_SPLIT_V:
+      property.inputs = {{0, {}}};
+      property.outputs = {{0, {}}};
+      property.restrict_same_input_output_scale = true;
```
Comment (akarmi) [inline tensorflow/lite/tools/optimize/operator_property.cc]: Should property.arbitrary_outputs be set here as well, similar to Split op above?

Comment (wwwind) [reply tensorflow/lite/tools/optimize/operator_property.cc]: thanks for spotting this! corrected.

Comment (akarmi) [reply tensorflow/lite/tools/optimize/operator_property.cc]: Thanks!

**Comment by jdduke** [comment]
We need to bump the op version for this.

To do this, you can add [logic in GetBuiltinOperatorVersion to detect the input/output types](https://github.com/tensorflow/tensorflow/blob/306dee4096e97520b1218d78c23b1a72941a9508/tensorflow/lite/tools/versioning/op_version.cc), and use an incremented version, update the op version to TFlite version mapping [here](https://github.com/tensorflow/tensorflow/blob/306dee4096e97520b1218d78c23b1a72941a9508/tensorflow/lite/toco/tflite/op_version.cc), and then update the max version where the builtin is registered [here](https://github.com/tensorflow/tensorflow/blob/306dee4096e97520b1218d78c23b1a72941a9508/tensorflow/lite/kernels/register.cc#L43).

**Comment by wwwind** [comment]
Hi @jdduke I bumped a version for this operator as requested. Please take a look. Thanks for the review!

**Review by jdduke — APPROVED** [review]

**Comment by wwwind** [comment]
@gbaned Can it be merged?

**Comment by wwwind** [comment]
Hi @rthadur ! Can this PR be merged ? It has been marked as 'ready_to_pull' many days ago. Thanks!

**Comment by rthadur** [comment]
@wwwind it got stuck internally , @jdduke can you please help merge this PR internally.

**Pull request merged**